htmresearch/support/neural_correlations_utils.py

```diff
@@ -489,4 +521,36 @@ def inputAnalysis(allSequences, model, numColumns):
   
   return overlapMatrix  
   
-  
\ No newline at end of file
+
+def computePWCorrelationsWithinCol(spikeTrains, removeAutoCorr, cellsPerColumn):
+  """
+  Computes pairwise correlations from spikeTrains
+  
+  @param spikeTrains (array) spike trains obtained from the activation of cells in the TM
+     the array dimensions are: numCells x timeSteps
+  @param removeAutoCorr (boolean) if true, auto-correlations are removed by substracting
+     the diagonal of the correlation matrix
+  @param cellsPerColumn (int)
+  @return corrMatrix (array) numCells x numCells matrix containing the Pearson correlation
+      coefficient of spike trains of cell i and cell j
+  @return numNegPCC (int) number of negative pairwise correlations (PCC(i,j) < 0)
+  """
+  numCells = np.shape(spikeTrains)[0]
+  numCols = numCells / cellsPerColumn
```

Comment:
Why do I need to know cellsPerColumn or numCols? It seems that it won't affect the result corrMatrix